man/systemd.network.xml

```diff
@@ -706,6 +706,12 @@
             <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>OnDevice=</varname></term>
+          <listitem>
+            <para>A boolean. Specifies whether to take device as default route.</para>
```

Comment:
I figure this is useful for PPP devices .
http://stackoverflow.com/questions/8178093/how-to-find-the-default-gateway-of-ppp-interface